tests/params/basic.yml

```diff
@@ -56,7 +58,8 @@ demographics:
       prep:
         adherence: 0.1
         discontinue: 0.1
-        coverage: 0.3
+        target: 0.3
+        init: 0.5
```

Comment:
why have this init higher than target?

Reply:
I wanted to check that the init value is what's hitting when I use that.